src/config/babelrc.js

```diff
@@ -42,8 +41,12 @@ module.exports = {
     isUMD
       ? require.resolve('babel-plugin-transform-inline-environment-variables')
       : null,
+    // TODO: use loose mode when upgrading to babel@7
     require.resolve('babel-plugin-transform-class-properties'),
     require.resolve('babel-plugin-transform-object-rest-spread'),
     require.resolve('babel-plugin-minify-dead-code-elimination'),
+    !treeshake
+      ? require.resolve('babel-plugin-transform-es2015-modules-commonjs')
```

Comment:
modules-commonjs also has a `loose` mode, but i dont think its worth using as it breaks namespace imports

```js
import * as namespace from './namespace'
```

in `loose` namespace would have **enumerable** `__esModule` property which can surprise quite a bit when spreading a namespace